### ISSUE TITLE
Add support for prefix and suffix on @JsonAnySetter annotation

### ISSUE THREAD

**Issue by alxxyz** [FasterXML/jackson-annotations#202]
In our POJO we have 2 `Map`s and both have keys prefixes.
So it will very help if @JsonAnySetter annotation will support some prefixes and suffixes.

**Comment by cowtowncoder** [comment]
Contributions welcome!

**Comment by alxxyz** [comment]
@cowtowncoder as I can see in this repository there are only the annotations, but where I can find the annotations processors?
As I understand I need to and also tests for that module?

**Comment by cowtowncoder** [comment]
@alxxyz Yes, `jackson-databind` has the annotation introspectors. Testing would probably require local snapshot builds.

**Comment by slovdahl** [comment]
Just to add some more context, this would also be helpful as an alternative to `JsonUnwrapped` not working for `Map<T, V>` (#171). Right now it's easy to _serialize_ POJOs using `@JsonAnyGetter` but _deserializing_ is not as safe because `@JsonAnySetter` would put any extra fields not matching any other annotation in the same `@JsonAnySetter` map.
